components/aws/sagemaker/tests/unit_tests/tests/common/test_boto3_manager.py

```diff
@@ -0,0 +1,50 @@
+import unittest
```

Comment:
How is the unit test coverage looking before vs after ? (Again I might have missed if this was already discussed some place)

Reply:
```
========================================================== 134 passed, 2 warnings in 3.00s ===========================================================
Name                                                      Stmts   Miss  Cover   Missing
---------------------------------------------------------------------------------------
batch_transform/src/sagemaker_transform_component.py         69     10    86%   78, 101, 142, 151-157, 165-170
batch_transform/src/sagemaker_transform_spec.py              45      0   100%
common/boto3_manager.py                                      31      4    87%   41, 124-126
common/common_inputs.py                                      34      0   100%
common/component_compiler.py                                 69     17    75%   80-86, 106-112, 229-253, 272, 298
common/sagemaker_component.py                               186      5    97%   64, 150, 199, 270, 275
common/sagemaker_component_spec.py                           51      3    94%   93, 187, 191
common/spec_input_parsers.py                                 30      9    70%   29-32, 37, 40-41, 48, 56
deploy/src/sagemaker_deploy_component.py                    119     43    64%   105-106, 111, 147, 209-213, 228-241, 256-264, 275-283, 294-303, 314-324, 328-333
deploy/src/sagemaker_deploy_spec.py                          47      0   100%
ground_truth/src/sagemaker_ground_truth_component.py        120     26    78%   101, 153, 170, 181, 187, 196-210, 227-234, 238-246, 323, 332-335, 343-348
ground_truth/src/sagemaker_ground_truth_spec.py              53      0   100%
hyperparameter_tuning/src/sagemaker_tuning_component.py     122      9    93%   89, 275, 284-292, 316-321
hyperparameter_tuning/src/sagemaker_tuning_spec.py           57      0   100%
model/src/sagemaker_model_component.py                       51      8    84%   63-64, 106, 115, 119-124
model/src/sagemaker_model_spec.py                            36      0   100%
process/src/sagemaker_process_component.py                   78     14    82%   77, 123, 129-130, 133, 149, 158, 167, 176-177, 200-205
process/src/sagemaker_process_spec.py                        43      0   100%
train/src/built_in_algos.py                                   1      0   100%
train/src/sagemaker_training_component.py                   138     14    90%   103, 143-146, 159, 176, 179, 292, 301-307, 315-320
train/src/sagemaker_training_spec.py                         47      0   100%
workteam/src/sagemaker_workteam_component.py                 36      5    86%   85, 109-114
workteam/src/sagemaker_workteam_spec.py                      30      0   100%
---------------------------------------------------------------------------------------
TOTAL                                                      1493    167    89%
```